app/src/main/java/org/p2p/wallet/newsend/ui/search/NewSearchPresenter.kt

```diff
@@ -192,7 +202,12 @@ class NewSearchPresenter(
         view?.showNotFound()
     }
 
+    private fun showSkeleton(show: Boolean) {
```

Comment:
isShow / isVisible / isLoading
and i suggest to remove implementations details in method name and rename it with `showLoading`